lib/cn/csched_sp3_work.c

```diff
@@ -76,30 +76,28 @@ sp3_work_estimate(struct cn_compaction_work *w)
     dst_is_leaf = false; /* TBD below */
 
     switch (w->cw_action) {
-
-        case CN_ACTION_NONE:
-        case CN_ACTION_END:
-        case CN_ACTION_SPLIT:
-            break;
-
-        case CN_ACTION_COMPACT_K:
-            /* Assume no garbage collection, thus percent_keep == 100 */
-            consume = halen + kalen;
-            dst_is_leaf = src_is_leaf;
-            break;
-
-        case CN_ACTION_COMPACT_KV:
-            consume = halen + kalen + valen;
-            percent_keep = 100 * 100 / cn_ns_samp(&w->cw_ns);
-            dst_is_leaf = src_is_leaf;
-            break;
-
-        case CN_ACTION_SPILL:
-            assert(cn_node_isroot(w->cw_node));
-            consume = halen + kalen + valen;
-            percent_keep = 100 * 100 / cn_ns_samp(&w->cw_ns);
-            dst_is_leaf = true;
-            break;
+    case CN_ACTION_NONE:
+    case CN_ACTION_SPLIT:
```

Comment:
TODO for split?

Reply:
Yeah, it does seem like you should do something here for split :-)